lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -4754,6 +4777,7 @@ class DecomposeComplexOpsPass
     addPatternIfTargetOpIsIllegal<DecomposeAtenScalarTensor>(patterns);
     addPatternIfTargetOpIsIllegal<DecomposeAtenScatterValueOp>(patterns);
     addPatternIfTargetOpIsIllegal<DecomposeAtenSignOp>(patterns);
+    addPatternIfTargetOpIsIllegal<DecomposeAtenTypeAsOp>(patterns);
```

Comment:
you should also add this in:  https://github.com/llvm/torch-mlir/blob/38fb99df658faf46e1266a684b0900d183167371/lib/Dialect/Torch/Transforms/LowerToBackendContract.cpp#L364

Reply:
Done!